assembly/src/assembler/context.rs

```diff
@@ -264,7 +265,7 @@ impl ProcedureContext {
         self.visibility.is_syscall()
     }
 
-    pub fn into_procedure(self, code: CodeBlock) -> Box<Procedure> {
+    pub fn into_procedure(self, code: MastNodeId) -> Box<Procedure> {
```

Comment:
minor nit: should we change the name of the `code` parameter to something like `body_node_index`?

Reply:
Agreed. Changed it everywhere in `Procedure` as well.